### DISCUSSION TITLE
Insights : of to set custom range in a time series

### DISCUSSION THREAD

**Discussion by neo7-studio-web** [directus/directus#10619] [Archive]
I'd like to see the number of articles I published by month during the past months.
I managed to do it well : 
![ins1](https://user-images.githubusercontent.com/18073828/146825014-e44b8c85-22ea-4c35-be61-6c2c395769f0.jpg)

But I cannot select a range wider than 3 months. If I try to type "Past 10 Months" in "Other" the graph is empty
![ins2](https://user-images.githubusercontent.com/18073828/146825143-5bc50998-3eff-42c8-b97c-88a361fe6656.jpg)

Is there is a correct way to add a wider range in time series ?

**Comment by rijkvanzanten** [comment]
Heya! The value you enter there will be passed to the API's `$NOW()` dynamic variable. Try setting it to `-10 months`.

**Reply by neo7-studio-web** [reply]
It worked, thanks a lot Rijk !
For information, it has to be `10 months` without the minus sign.
